src/classes/TDTM_DefaultConfig.cls

```diff
@@ -194,11 +194,6 @@ public without sharing class TDTM_DefaultConfig {
               Class__c = 'PMT_Payment_TDTM', Load_Order__c = 1, Object__c = 'Opportunity',
               Trigger_Action__c = 'BeforeUpdate;AfterInsert;AfterUpdate'));
 
-        // Payment records - currency code validation
```

Comment:
The code was introduced in a Work Item prior to this one and should stay.

Reply:
Interesting, I'll have to look into why this class was modified as part of this branch. Has this code been merged into master?

Reply:
It is in the master. It must be something with the local copy that is mismatched?

Reply:
Thanks, I'll pull the copy from master.